cohortextractor/emis_backend.py

```diff
@@ -541,7 +543,10 @@ def patients_with_these_medications(self, **kwargs):
             # Remove unhandled arguments and check they are unused
             assert not kwargs.pop("episode_defined_as", None)
             return self._patients_with_events(
-                MEDICATION_TABLE, "", "snomed_concept_id", **kwargs,
```

Comment:
If you're keen to keep this line (and other similar ones) as close to the original as before then you could remove the final comma instead.  A recent black release changed trailing commas to force one item per line.

Reply:
Thanks.  I'm not really fussed tbh.